packages/flutter_localizations/lib/src/l10n/material_en.arb

```diff
@@ -207,7 +207,7 @@
     "description": "The label for the Search Web button and menu items on iOS."
   },
 
-  "shareButtonLabel": "Share...",
+  "shareButtonLabel": "Share",
   "@shareButtonLabel": {
     "description": "The label for the Share button and menu items on iOS."
```

Comment:
The description is inaccourate. same for other buttonlabel in this file

Reply:
I upated the PR to remove the inaccurate 'on iOS' reference in those button label descriptions.